arrow_pd_parser/export.py

```diff
@@ -67,3 +71,30 @@ def pd_to_json(
             new[col].replace("NaT", np.nan, regex=False, inplace=True)
 
     new.to_json(output_file, orient=orient, lines=lines, **kwargs)
+
+
+def pd_to_parquet(
+    df: pd.DataFrame,
+    output_file: Union[str, pa.lib.NativeFile],
+    from_pandas_kwargs={},
+    write_table_kwargs={},
+    arrow_schema: pa.lib.Schema = None,
+):
+    """
+    Export a data frame as parquet
+    Does no conversion, and adheres to the parquet schema
+
+    Args:
+        df (pd.DataFrame): a pandas dataframe
+        output_file str: the path you want to export to (s3)
+    """
+
+    if type(output_file) != str and type(output_file) != pa.lib.NativeFile:
+        raise TypeError(f"unsupported output type: {type(output_file)}")
+
+    table = pa.Table.from_pandas(df, **from_pandas_kwargs, schema=arrow_schema)
```

Comment:
Still no check if they supply `schema` in the `from_pandas_kwargs` and `arrow_schema`